go/ms/internal/validator/rpkivalidator.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package validator
```

Comment:
Seems a bit strange to have a separate package just for two variables.

How do you expect validation to be extended?

Reply:
The idea was that we can use an existing validator library if possible or build one to extend this. Right now we just use the script in that file